roles/upgrade/tasks/leapp-post-upgrade.yml

```diff
@@ -76,25 +80,26 @@
     - satellite_organization is defined
     - satellite_activation_key_post_leapp is defined
 
-- name: Include update-and-reboot.yml
+- name: leapp-post-upgrade | Include update-and-reboot.yml
   ansible.builtin.include_tasks: update-and-reboot.yml
 
 # TODO: Validate RHEL OS versions again?
 
 # Only found in RHEL 7 to 8 documentation
-- name:  Old kernels have been removed from the bootloader entry for RHEL 7 to 8
+- name: leapp-post-upgrade |  Old kernels have been removed from the bootloader entry for RHEL 7 to 8
   ansible.builtin.shell: |
-    export PATH={{ os_path }}
+    set -o pipefail;
```

Comment:
this change here would be the only concern that probably should be tested to ensure that the export path doesn't need to be first and we can set pipefail before it without issue. I do not think it should matter but in theory that is the main change that I could see breaking something unexpected